### DISCUSSION TITLE
Pet detection firmware for E1 Outdoor

### DISCUSSION THREAD

**Discussion by furymeal** [AT0myks/reolink-fw-archive#16] [Firmware sharing]
i got the 804.22012406 FW from the support
HW Version IPC_523SD8
Support said it contains the detction for animals

https://support-d.reolink.com/attachments/token/3FVDyr7FnfXKiHrbI3ihfGwzW/?name=IPC_523SD8.804_22012406.E1-Outdoor.OV05A10.5MP.WIFI1021.PTZ.REOLINK%28open_ai_trace%29.pak

**Comment by furymeal** [comment]
pet detection and tracking works great

**Comment by AT0myks** [comment]
Added, thanks. Do the more recent firmwares not support animal detection?

**Reply by furymeal** [reply]
No, Reolink doesnt even mention, that pet detection was supported at some point. Officialy 5 MP cameras are not compatible with pet detection and never will be
